go.mod

```diff
@@ -174,3 +174,5 @@ require (
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+replace gopkg.in/vansante/go-ffprobe.v2 => github.com/mjh1/go-ffprobe v0.0.0-20230330150500-3b664b71952e
```

Comment:
Did you raise a PR against the upstream repo?

Reply:
not yet as i was planning on expanding it to include all the different side data. but through a bit of searching it looks like there may be too much to include in named struct fields :( can't find a definitive list of all variations.
Found this typescript version: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/ffprobe/index.d.ts

But then chatgpt says there are way more options ha:
<img width="730" alt="Screenshot 2023-03-31 at 12 24 06" src="https://user-images.githubusercontent.com/3630965/229107708-27692b9d-13df-40f1-aa6c-c8a1e2f6cd48.png">

So might have to be a `map[string]interface{}` kind of thing

Reply:
Yeah, that sounds like a good idea and then we can leave it to someone else to do the full list if needed